M2/Macaulay2/m2/mathml.m2

```diff
@@ -169,14 +172,13 @@ mathML GradedModuleMap :=
 mathML GroebnerBasis :=
 mathML IndexedVariableTable :=
 mathML Package :=
-mathML Resolution :=
```

Comment:
The documentation for `Resolution` says:
> These resolutions are internal engine objects not meant to be examined by the user.

Eventually I'd like to remove `Resolution` and use `ResolutionComputation` only.

Reply:
If you make any changes to Resolution, or plan to, I'd like to know, as I will be also making changes, to simplify that code (we could do it together too), to link it with Complexes.m2 (after the 1.18 release, and that will only be on a branch for the moment).

Reply:
I asked you about this in the top comment of the pull request:

> Possible things to do in a future PR:
> - replace the `Resolution` type with `ResolutionComputation`
> - move res.m2 out of Core and into Complexes.m2, then add Complexes to the preloaded packages. I know this is possible, because after the last commit I realized that no other sources in the Core depend on `res.m2`.
> 
> @mikestillman let me know what you think of the last two items.